app/controllers/admin/user_creations_controller.rb

```diff
@@ -1,46 +1,69 @@
 class Admin::UserCreationsController < ApplicationController
   
   before_filter :admin_only
+  before_filter :get_creation
+  
+  def get_creation
+    @creation_class = params[:creation_type].constantize
+    @creation = @creation_class.find(params[:id])
+  end
   
   # Removes an object from public view
   def hide
-    creation_class = params[:creation_type].constantize
-    creation = creation_class.find(params[:id])
-    creation.hidden_by_admin = (params[:hidden] == "true")
-    creation.save(:validate => false)
-    action = creation.hidden_by_admin? ? "hide" : "unhide"
-    AdminActivity.log_action(current_admin, creation, action: action)
-    flash[:notice] = creation.hidden_by_admin? ? 
+    @creation.hidden_by_admin = (params[:hidden] == "true")
+    @creation.save(:validate => false)
+    action = @creation.hidden_by_admin? ? "hide" : "unhide"
+    AdminActivity.log_action(current_admin, @creation, action: action)
+    flash[:notice] = @creation.hidden_by_admin? ? 
```

Comment:
Avoid multi-line ?: (the ternary operator); use `if`/`unless` instead.<br>Trailing whitespace detected.